src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/court/commonplatform/CommonPlatformCourtEventListenerIntTest.kt

```diff
@@ -289,29 +243,6 @@ class CommonPlatformCourtEventListenerIntTest : MessagingMultiNodeTestBase() {
     )
   }
 
-  @Test
-  fun `should process large messages`() {
-    stubPersonMatchScores()
-    stubPersonMatchUpsert()
-    val defendantId = randomDefendantId()
-
-    val s3Key = UUID.randomUUID().toString()
-
-    val request =
-      PutObjectRequest.builder().bucket(s3Bucket).key(s3Key).build()
-
-    s3AsyncClient.putObject(request, AsyncRequestBody.fromString(commonPlatformHearing(listOf(CommonPlatformHearingSetup(defendantId = defendantId))))).get()
-    val messageId = publishLargeCommonPlatformMessage(
-      largeCommonPlatformMessage(s3Key, s3Bucket),
-    )
-
-    checkTelemetry(
-      MESSAGE_RECEIVED,
-      mapOf("MESSAGE_ID" to messageId, "SOURCE_SYSTEM" to COMMON_PLATFORM.name, "EVENT_TYPE" to COMMON_PLATFORM_HEARING.name),
-    )
-    awaitNotNullPerson { personRepository.findByDefendantId(defendantId) }
-  }
```

Comment:
covered by `should publish large message to CPR court topic including the cpr uuid`